### DISCUSSION TITLE
Maybe add an option for 2.0 style pallet?

### DISCUSSION THREAD

**Discussion by janSipiki** [ScratchAddons/ScratchAddons#2112] [Ideas]
I prefer the old style pallet from the new one, as it is more compact and easy to use. Maybe try mimicking the pallet from this old Scratch 3 mockup?
https://en.scratch-wiki.info/w/images/archive/20170622081100%21Scratch_3.0_Editor_Preview_1.png

Edit: For extensions and other extra categories, maybe add a button to go to a second page of icons? Like, an left and right arrow to see other categories.

**Comment by qbjl** [comment]
Why do I love this? 

I'd love there to be an option for this type of styling for the editor. If the creators of Scratch Addons can make custom icons for block palettes, then they can probably do this.

THANKS FOR THIS!

**Reply by retronbv** [reply]
> If the creators of Scratch Addons can make custom icons for block palettes, then they can probably do this.

That was fairly easy iirc, it's only injecting an image to the DOM. This would be considerable harder

**Reply by janSipiki** [reply]
> > If the creators of Scratch Addons can make custom icons for block palettes, then they can probably do this.
> 
> That was fairly easy iirc, it's only injecting an image to the DOM. This would be considerable harder

I mean, you'd only need to move the icons up. How hard could it be?

**Reply by nexensys** [reply]
VERY

**Comment by just-a-hriday** [comment]
Yeah, this does actually look really nice.

**Comment by cobaltt7** [comment]
The main problem is compatibility with hide-flyout. how would that work?
________________________________

**Reply by WorldLanguages** [reply]
I wouldn't say that's the "main" problem, tbh

**Reply by nexensys** [reply]
Just have some `!important` css override it. I will try this soon.

**Comment by cobaltt7** [comment]
Well there are blockly functions that do everything else (someone shared in the discord a while back)
________________________________
